pallets/proposals/src/benchmarking.rs

```diff
@@ -18,37 +18,39 @@ benchmarks! {
         T::AccountId: AsRef<[u8]>,
     }
 
-    update_project {
-        let milestones: BoundedProposedMilestones = vec![ProposedMilestone {
-            percentage_to_unlock: 1,
-        }; 100].try_into().unwrap();
+    create_project {
+        let caller: T::AccountId = whitelisted_caller();
 
-        let caller = create_project_common::<T>(u32::MAX.into());
+        let milestones: BoundedProposedMilestones<T> = vec![ProposedMilestone {
```

Comment:
Again we iterate over proposed milestones in the extrinsic, so this needs to be the maximum amount of proposed milestones.